api/swagger-spec/v1.json

```diff
@@ -4374,18 +4820,12 @@
       "consumes": [
        "*/*"
       ]
-     }
-    ]
-   },
-   {
-    "path": "/api/v1/watch/persistentvolumes",
-    "description": "API at /api/v1",
-    "operations": [
+     },
      {
-      "type": "json.WatchEvent",
-      "method": "GET",
-      "summary": "watch individual changes to a list of PersistentVolume",
-      "nickname": "watchNamespacedPersistentVolumeList",
+      "type": "unversioned.Status",
+      "method": "DELETE",
+      "summary": "delete collection of PersistentVolume",
+      "nickname": "deletecollectionNamespacedPersistentVolume",
```

Comment:
same here, a PV is global I thought?  Only a PVC is namespaced if I recall.  Why is this saying namespaced